views/update/overseas-entity-query.html

```diff
@@ -0,0 +1,35 @@
+{% extends "layout.html" %}
+
+{% set title = "What is the OE number? (NOT LIVE)" %}
+
+{% block pageTitle %}
+{% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+{% include "includes/back-link.html" %}
```

Comment:
I do not know where the back link goes

Reply:
It refreshes the same page, similar to the existing landing page on the register journey.

Reply:
The existing landing page is more complex, if you do not what to add just remove the backLink block please

Reply:
@mouhajer-ch how do I remove the back button haha...
I've tried removing the block from the view, I have also tried adding the below to the view:

`{% block noBackLink %}
{% include "includes/no-back-link.html" %}
{% endblock %}`

But the back link still appears whenever I view the page

Reply:
Override with an empty block, something like the `{% block backLink %}{% endblock %}` should do

Reply:
This should be ok now @mouhajer-ch if you want to double-check for me.